MapleServer2/MapleServer.cs

```diff
@@ -29,13 +30,14 @@ public static void Main()
             CultureInfo.CurrentCulture = new CultureInfo("en-US");
 
             // Load .env file
-            string dotenv = Path.Combine(Paths.SOLUTION_DIR, ".env");
+            string filePath = Path.Combine(Paths.SOLUTION_DIR, ".env");
```

Comment:
Isn't `filePath` less descriptive than `dotenv`?

Reply:
yeah you're right